.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+  - repo: local
+    hooks:
+      - id: black
+        name: black
+        language: system
+        entry: black
+        types: [python]
+      - id: isort
```

Comment:
Is there a specific reason run isort after black?

Reply:
isort has more opinions than black, but black is ok with what isort does. Due to isort respecting black formatting it's easier to run it after black since isort has some behaviour that black doesn't like when it encounters blackend code.